### DISCUSSION TITLE
Consider using Google Java Format with Git Plugin

### DISCUSSION THREAD

**Discussion by motofix** [g0ddest/fixedlength#11] [Ideas]
The idea would be to use the [Git Code Format Plugin](https://github.com/Cosium/git-code-format-maven-plugin) that automatically format your code on commit.

This is the guarantee to have an always good-looking well formatted coded without any pain for the developers.

**Comment by g0ddest** [comment]
There is a checkstyle plugin with a config here: https://github.com/g0ddest/fixedlength/tree/master/config/checkstyle

**Comment by motofix** [comment]
The checkstyle plugin doesn't automatically format the code on commit, is it ?

**Reply by g0ddest** [reply]
It's checking style on every build
